plugin/src/main/java/com/denizenscript/denizen/events/entity/AreaEffectCloudApplyScriptEvent.java

```diff
@@ -0,0 +1,63 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.AreaEffectCloudApplyEvent;
+
+public class AreaEffectCloudApplyScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // area effect cloud applies
+    //
+    // @Group Entity
+    //
+    // @Cancellable true
+    //
+    // @Triggers when an area_effect_cloud tries to apply its effect(s) to entities within range.
+    //
+    // @Warning
+    // This runs every 5 ticks if there are any entities in the area effect cloud's bounding box. Prefer <@link event entity potion effects modified> for listening to normal potion effect changes.

```

Comment:
It's documented as `area_effect_cloud` in `Triggers` and as `area effect cloud` in the warning - personally I think `area effect cloud` is better as it's just explaining something in English, not directly referencing some other doc or anything, but either way should be consistent across the meta

Reply:
This was my suggestion and was intentional - you should slip multiple variants of the same text into documentation when possible to improve searchability (or use `@Synonyms` when it can't be naturally added)